.travis.yml

```diff
@@ -18,10 +18,10 @@ matrix:
     services:
       - docker
     script:
-      - ./gradlew -s -i check && ./gradlew_win -s -i check
+      - ./gradlew -s -i check build
```

Comment:
`./gradlew_win` is missing (to test windows)

Reply:
In fact this would be probably better (so windows is tested in parallel):

```
matrix:
  include:
  - os: linux
    script:
      - ./gradlew -s -i check build
  - os: linux
    sudo: required
    services:
      - docker
    script:
      - ./gradlew_win -s -i check build
  - os: osx
    script:
      - ./gradlew -s -i check build
```

Reply:
👍